src/index.ts

```diff
@@ -2,7 +2,7 @@ import { translate } from "./engine";
 import { Schema } from "./schema";
 import { Schemas } from "./schemas";
 
-export default {
+export = {
```

Comment:
to make it possible to import iuliia in Node like this:

```js
const iuliia = require('iuliia');
```

instead of this:
```js
const iuliia = require('iuliia').default;
```

Tested with TypeScript, the following construction will still work:

```ts
import iuliia from 'iuliia';
```